sql/branch.sql

```diff
@@ -0,0 +1,3 @@
+BEGIN;
+    UPDATE payment_instructions SET due = 0 WHERE due > 9.41;
```

Comment:
> For the fifth, amount = 0, meaning due should be reset to zero.

Done in https://github.com/gratipay/gratipay.com/pull/3981/commits/cc1d5f10a8cd89b276543309534490e43996f2af